.coveragerc

```diff
@@ -48,6 +48,7 @@ omit =
     homeassistant/components/downloader.py
     homeassistant/components/keyboard.py
     homeassistant/components/light/hue.py
+    homeassistant/components/light/mqtt.py
```

Comment:
Yes, the given switch example should help with it. For example, if
brightness topic given, return last value on topic else None

On Wed, Oct 28, 2015, 01:56 hexxter <notifications@github.com> wrote:

> In .coveragerc
> <https://github.com/balloob/home-assistant/pull/552#discussion_r43229829>:
>
> > @@ -48,6 +48,7 @@ omit =
> >      homeassistant/components/downloader.py
> >      homeassistant/components/keyboard.py
> >      homeassistant/components/light/hue.py
> > +    homeassistant/components/light/mqtt.py
>
> ok i will write the unittest. What is with the comment from @tomduijf
> <https://github.com/tomduijf> ? Should i rewrite my code with one class?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant/pull/552/files#r43229829>.
>